### PR TITLE
Wire tls-sni-cert-key flag

### PR DESCRIPTION
**Pull request by soltysh**
/assign @deads2k

### CODE REVIEW

File: pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/flags.go [hunk 1]
```diff
@@ -278,3 +278,11 @@ func admissionFlags(admissionPluginConfig map[string]configv1.AdmissionPluginCon
 
 	return args, nil
 }
+
+func sniCertKeys(namedCertificates []configv1.NamedCertificate) []string {
+	args := []string{}
+	for _, nc := range namedCertificates {
+		args = append(args, fmt.Sprintf("%s,%s:%s", nc.CertFile, nc.KeyFile, strings.Join(nc.Names, ",")))
```
Comment (sttts) [inline pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/flags.go]: have you checked that we support `foo,bar:` without the names?

Comment (soltysh) [reply pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/flags.go]: Yes, it'll parse this as empty, see here: https://github.com/openshift/origin/blob/2b961cd33770c2593dc31e9d1d2a73bdd70da631/vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/util/flag/namedcertkey_flag.go#L44-L53

**Pull request merged**